course/lesson-3-standing-on-shoulders-of-giants/ExampleRotateSprite.tscn

```diff
@@ -16,7 +29,10 @@ margin_left = 7.0
 margin_top = 7.0
 margin_right = 1040.0
 margin_bottom = 291.0
+<<<<<<< HEAD
 scene = ExtResource( 1 )
+=======
+>>>>>>> main
```

Comment:
This merge conflict text will prevent the scene from loading

Reply:
Ok will fix